src/pages/blog/videos/_all-videos.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import VideoSrc from '../../markets/static/video/globe.mp4'
+import VideoCard from './_video-card'
+import VideoPlayer from './_video-player'
+import { Container, Flex } from 'components/containers'
+import { Text, LocalizedLinkText } from 'components/elements'
+import { localize } from 'components/localization'
+import RightArrow from 'images/svg/black-right-arrow.svg'
+
+const VideoGrid = styled.div`
+    display: grid;
+    width: 100%;
+    height: 100%;
+    margin: 8rem 0;
+    grid-template-columns: repeat(auto-fit, minmax(288px, 384px));
+    grid-row-gap: 40px;
+    grid-column-gap: 24px;
+    grid-template-rows: auto;
+`
+
+const AllVideos = ({ video_data }) => {
+    const [show, setShow] = useState(false)
+
+    const handleCloseVideo = () => setShow(false)
+    const handleOpenVideo = () => setShow(true)
```

Comment:
```suggestion
```
since this seems to be only used in one place each, i think we should just pass the function directly to the props instead of declaring a function here